clang/include/clang/Driver/Options.td

```diff
@@ -3596,6 +3596,9 @@ def fopenmp : Flag<["-"], "fopenmp">, Group<f_Group>,
   Flags<[NoArgumentUnused]>,
   Visibility<[ClangOption, CC1Option, FlangOption, FC1Option]>,
   HelpText<"Parse OpenMP pragmas and generate parallel code.">;
+def fopenmp_default_none : Flag<["-"], "fopenmp-default-none">, Group<f_Group>,
+  Visibility<[FlangOption, FC1Option]>,
```

Comment:
Do you mind moving this to later in this file? Maybe so it is before `def fopenmp_use_tls`. There is a loose ordering of options here with the "top-level" `fopenmp` options first and some more specific ones later. This file is already huge and somewhat disorganized, so it would be good to keep what organization there is.

Reply:
Done. I have seen that there seems to be a bit of chaos (or: I did not the see the order :-)) in the order of the CLI options. Maybe it warrants an NFC PR that puts the options in some better order.